index.html

```diff
@@ -14,8 +14,8 @@
   <body>
     <header class="header">
       <div class="container">
-        <div class="item-1"><img src="images/My Logo.png" /></div>
-        <div class="item-2"><img src="images/Hamburguer.png" /></div>
+        <div class="item-1"><img src="images/MyLogo.png" alt="Logo" /></div>
+        <div class="item-2"><img src="images/Hamburguer.png" alt="Menu" /></div>
       </div>
     </header>
     <section>
```

Comment:
- [ ] By following the template design I found out that the paragraph is at the center but yours is still not yet centered. 
![image](https://user-images.githubusercontent.com/65068771/148326957-ecdf4f88-4ea8-44c7-8b9f-50d13e2006e4.png)

- [ ] The requirements say that The section uses the font and colors (background and font) defined for the template. Kindly let's go ahead put the background in this section and use the same `font-family`. You can find this background and the `font-family`in the Figma template. Check here 👇  how I can compare your design

Your design | The template 
------------ |-------------
![image](https://user-images.githubusercontent.com/65068771/148327448-99c403c4-dee7-4f9c-828b-6955ed8ff334.png) | ![image](https://user-images.githubusercontent.com/65068771/148327515-02649867-0850-47b5-b435-cb8f265359f6.png)

Reply:
Please ensure you stick to the design as much as possible (e.g. font, colors, images, tests, margins) using the templates in Figma.

### Your design	                                                                                                 
> 	
>![your-design](https://user-images.githubusercontent.com/10332499/148395342-1378ecfb-b0d3-490c-ba06-943c318592ac.PNG) 
### The template
![image](https://user-images.githubusercontent.com/65068771/148327515-02649867-0850-47b5-b435-cb8f265359f6.png)